### PR TITLE
Remember window position on screen

### PR DESCRIPTION
**Pull request by JATIN2111999**
Created function for Window position restore PR

### CODE REVIEW

**Comment by dreua** [comment]
Err you understood this the wrong way around, the rest of your commit message title was great I just wanted the number / link in the commit message body.

**Comment by JATIN2111999** [comment]
Now

**Comment by JATIN2111999** [comment]
hey @dreua is my PR right or wrong i am still confuse can you give any conformation i resolve all changes in single commit and also added whitespaces after commas.

File: pdfarranger/config.py [hunk 1]
```diff
@@ -111,6 +111,13 @@ def set_window_size(self, size):
         self.data.set('window', 'width', str(size[0]))
         self.data.set('window', 'height', str(size[1]))
 
+    def set_position(self,position):
```
Comment (dreua) [inline pdfarranger/config.py:114]: Here is still a space miissing

**Review by dreua — CHANGES_REQUESTED** [review]

**Comment by dreua** [comment]
The commit message I had in mind was: 
```
Remember window position on screen

Closes: https://github.com/pdfarranger/pdfarranger/issues/397
```
(Including the empty line)

File: pdfarranger/config.py (hunk 1 above)
Comment (JATIN2111999) [reply pdfarranger/config.py:114]: oh damn sorry

Comment (JATIN2111999) [reply pdfarranger/config.py]: done

**Comment by kbengs** [comment]
There is one line over 100 characters. Codacy use to compalin about that. Don't know why it doesn't now.
Maybe a bit late to comment but is this something we should do? Should the application itself decide where it will open up? Or is that something that the desktop environment should handle..? On KDE I see no other application that will remember it's position.

**Comment by dreua** [comment]
Most applications I just tested on Gnome remember their position: Firefox, Signal, Telegram Spotify
The Gnome applications like Terminal and Nautilus however, don't. Seems like it's up to the application.
Actually I don't care but I think it doesn't hurt and we can close this old issue with this PR. But we could of course also close it without this PR.

**Comment by JATIN2111999** [comment]
Yes you are right about that but it is necessary in some use cases you want your application to take that some place where you closed it so it can be helpful to navigate your mouse cursor and that's why @dreua created this issue.

**Comment by dreua** [comment]
> @dreua created this issue.

I did not do such a thing.

**Comment by JATIN2111999** [comment]
> > @dreua created this issue.
> 
> I did not do such a thing.

Oh yeah right sorry

**Comment by dreua** [comment]
> There is one line over 100 characters.

Yeah that looked long to me, too. Wasn't sure whether we break at 80 or 100 characters or not at all. If we want to enforce this, we need this checked by CI and documented, I'll merge this for now. Thanks @JATIN2111999 !

**Comment by dreua** [comment]
> it can be helpful to navigate your mouse cursor.

Wait, now I'm curious: Are you automating PDF Arranger with scripted mouse movement?

**Comment by JATIN2111999** [comment]
> > it can be helpful to navigate your mouse cursor.
> 
> Wait, now I'm curious: Are you automating PDF Arranger with scripted mouse movement?

There is no mouse movement I am just storing the window co-ordinates and moving the window to the last terminated co-ordinate that's all.

**Comment by dreua** [comment]
In this PR, yes I'm aware of that, I read the code. My question was about your use case

**Comment by JATIN2111999** [comment]
> In this PR, yes I'm aware of that, I read the code. My question was about your use case

My OS don't remembered the position of window and also we are storing all properties like window size and acceleration this property is important to some extent.
So the use case will be like if we close the program by mistake we can open the application at the same place instead of finding the window in new place this may be very useful for big screen users as we are storing the window size we also giving them the exact position of window where it was terminated.

**Pull request merged**